devices/common_patches/merge_I2C_OAR1_ADDx_fields.yaml

```diff
@@ -0,0 +1,15 @@
+# Merge together I2C OAR1 wrongly separated ADDx fields
+
+"I2C*":
+  OAR1:
+    _modify:
+      ADD_0:
+        name: ADD0
+      ADD_1_7:
+        name: ADD7
+      ADD_8_9:
+        name: ADD9
+      ADD10:
+        name: ADD9
```

Comment:
Should this one be ADD10?

Reply:
No. It is done on purpose. 
Some devices have ADD_0 ADD_1_7 ADD_8_9 that will be merged to ADD other have ADD0 ADD7  and ADD10 that will be merged to ADD. I can't use `ADD*`  because the register has an ADDMODE field so I use ADD[019] as I haven't find a way to merge ADD0, ADD7, ADD9, ADD10 so I rename ADD10 to ADD9 as a workaround. Will had a comment.